### PR TITLE
Update UG and DG to reflect name change

### PR DESCRIPTION
**Pull request by AlekKwek18**

### CODE REVIEW

**Review by KengHian — APPROVED** [review]
LGTM.

File: docs/images/UndoRedoState1.png
Comment (Quasant) [inline docs/images/UndoRedoState1.png:1]: I understand the changes for the other png, but this one I noticed there was an extra ab2:AddressBook. 

Not sure if intended or not.

**Review by Quasant — APPROVED** [review]
Okay DG can still be changed during feature freeze, so should be fine.

File: docs/images/UndoRedoState1.png
Comment (Green-Tea-123) [reply docs/images/UndoRedoState1.png:1]: I think it's to standardise the diagrams (the subsequent image uses ab2)

**Review by Green-Tea-123 — APPROVED** [review]
LGTM.

**Pull request merged**